### ISSUE TITLE
Issue connecting desktop client to nextcloud in docker

### ISSUE THREAD

**Issue by tanja84dk** [nextcloud/docker#861]
Hello Nextcloud

I have a small issue where I cantt login to my nextcloud there is in a docker container from my desktop client, and there is no issue from my mobile client and also no issue within chrome browser.

after I have entered my password then its just looping with grant access.

Here is a small video of showing what is happening.
https://youtu.be/hP5UvtiqKAY

EDIT: just noticed that its in the second part of the video because obs made a issue where it did not stop the recording and start again when I used the hotkey for it

**Comment by J0WI** [comment]
I don't think that this issue is directly related to Docker. Have you already searched for this problem in https://github.com/nextcloud/desktop?

**Comment by aaronduty** [comment]
I also hit this. The browser dev tools provides the following when clicking "Grant Access":

> Content Security Policy: The page’s settings blocked the loading of a resource at http://_url_/login/v2/grant (“form-action https://_url_”).

I resolved it by adding `'overwriteprotocol' => 'https'` to the nextcloud config/config.php file. I believe I also had to change overwrite.cli.url as well.

I found it on an article at https://help.nextcloud.com/t/fixing-mixed-content-errors-when-using-a-proxy/13571 and I am still getting some CSP messages on the web frontend, but I may have messed a config up somewhere. I'll keep digging into that, but hopefully the above is helpful.
